### PR TITLE
Mouseover image saturation

### PR DESCRIPTION
**Pull request by ryantmer**
Applies a filter to course image on mouseover, to give us the colorization effect. This required switching to an `` rather than using `background-image`, which is something we wanted to do anyway.

### CODE REVIEW

**Comment by ryantmer** [comment]
@apalaniuk @d2l-joelp Just a little thing I had mostly done on Friday (but forgot to push (and then found bugs in (and then fixed today))).

File: d2l-course-tile-styles.html [hunk 1]
```diff
@@ -38,8 +38,27 @@
...
 			border-top-right-radius: 10px;
 			background: #B9C2D0;
-			background-position: center;
-			background-size: cover;
+			overflow: hidden;
+			z-index: 0;
+		}
+		.course-image img {
+			width: 100%;
+			position: absolute;
```
Comment (d2l-joelp) [inline d2l-course-tile-styles.html]: Do we need `position:absolute` and top, bottom, etc. on the course-image div itself in addition to the same styles on the `img`?

Comment (ryantmer) [inline d2l-course-tile-styles.html]: Turns out they are required, but the `:before` stuff can mostly go. I've pushed a commit, but GH is currently experiencing some issues.

Comment (ryantmer) [inline d2l-course-tile-styles.html]: Yay, GH has caught up now.

**Comment by d2l-joelp** [comment]
LGTM

**Pull request merged**